src/common/rdtsc.h

```diff
@@ -32,6 +36,13 @@ static inline u64 FencedRDTSC() {
 }
 #endif
 
+#else
+// ARM
+static inline u64 FencedRDTSC() {
+    return 0;
```

Comment:
Same here for actual implementation or TODO.

Reply:
Done